rskj-core/src/main/java/org/ethereum/core/genesis/GenesisLoaderImpl.java

```diff
@@ -230,6 +232,20 @@ private void setupPrecompiledContractsStorage(Repository repository) {
         }
     }
 
+    public static InputStream loadGenesisFile(String fileName) {
```

Comment:
Does it need to be public? Since you need only visible for testing, you can let it package private with the default access, wdyt?

Reply:
oh yea cool, wasn't aware of default access. thanks!
That makes me think about all the other "unit" tests that could be simplified and cleaner if we used default access...🧐